### PR TITLE
Update the manifest.toml main resources sources to tiki-26.0 which supports php8.1 to php8.2

### PR DESCRIPTION
**Pull request by rodriguezny**
## Problem

- *Tiki 26.0 is released and runs under php8.1 to php8.2, but the resources.sources.main in manisfest.toml is not yet update to support the installation of or upgrade to tiki-26.0

## Solution

- Update the manifest.toml main resources.sources and resources.apt packages versions

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by marclaporte** [comment]
@rodriguezny PHP 8.1 is well supported in Tiki 26x (we will fix any bug) but PHP 8.2 is Beta (We will fix everything for Tiki27 LTS): 
Ref: https://doc.tiki.org/Requirements#PHP_

**Comment by rodriguezny** [comment]
Hello @ericgaspar ! 
I have a blocker in the "resources.apt" section as described bellow:
Tiki26.0 (resources.main) and tiki-24.4 (resources.lts) require diverging versions of php.

tiki26.0 (resources.main) requires php8.1, so I updated the packages value to handle this as it is said in the doc that "when packages contains some phpX.Y-foobar dependencies, this will automagically define a phpversion setting equal to X.Y which can therefore be used in app scripts ($phpversion) or templates (__PHPVERSION__)". With this the installation works and upgrade works perfectly, even the app when testing after installation.

But tiki24.4 requires php7.4. With the current configuration (packages set to php8.1) the installation of tiki_ynh also succeeds but there are errors warnings in the log and also some fatal errors can appear during the app usage due the php version which is not satisfied.

So, my question is that, is there a way to specify the packages to install depending on the type of installation (longterm or regular) choosed ? I explored the yunohost doc but I found nothing that helped me. I also tried to use the syntax we use in the "resources.sources" section to specify where to get the app sources but it din't work.

Need help, thanks!

**Comment by ericgaspar** [comment]
What about using something like this?
```
    [resources.apt]
    packages = "tesseract-ocr imagemagick mariadb-server"

    packages_from_raw_bash = """
            if [[ "$release_cycle" == "regular" ]]; then 
                echo "php8.1-bz2 php8.1-curl php8.1-gd php8.1-zip php8.1-mysql php8.1-xml php8.1-mbstring php8.1-intl"  
            else
                echo "php7.4-bz2 php7.4-curl php7.4-gd php7.4-zip php7.4-mysql php7.4-xml php7.4-mbstring php7.4-intl"
            fi
        """
```

**Comment by rodriguezny** [comment]
Let me test it.

**Comment by rodriguezny** [comment]
Thank you @ericgaspar,  it works as expected.

File: manifest.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name = "Tiki"
 description.en = "Wiki-based, content management system"
 description.fr = "Système de gestion de contenu basé sur Wiki"
 
-version = "25.2~ynh2"
+version = "26.0~ynh2"
```
Comment (ericgaspar) [inline manifest.toml]:
```suggestion
version = "26.0~ynh1"
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Pull request merged**